dispatcher/backend/src/tests/integration/conftest.py

```diff
@@ -177,3 +171,12 @@ def pytest_sessionstart(session):
     before performing collection and entering the run test loop.
     """
     Initializer.check_if_schema_is_up_to_date()
+    with Session.begin() as session:
```

Comment:
It is true that not being able to detect left overs is an issue.
I added this because I used the DB for something else and it hit me in the face.
But you are true that this is probably not a good idea. Thank you.